Update CashRegister.cs

CCianfloneDev opened the pull request:
No longer throws exception if wrong item id is entered.
Removed unneeded block of code that converts list items to list of strings.

CCianfloneDev committed:
Update CashRegister.cs

No longer throws exception if wrong item id is entered.
Removed unneeded block of code that converts list items to list of strings.

DZ-FSDev commented on ColesStopAndShop/CashRegister.cs:
```diff
@@ -218,7 +218,15 @@ public void ProcessPayment(PaymentType paymentChosen)
-                    listOfItems.Add((ItemId)itemScanned);
+                    // This loop checks to verify the key exists in the pre-defined dictionary before adding it.
+                    // No error message is thrown if non existent and nothing is added to list, just moves on to next item.
+                    foreach (int item in storeDeployedAt.AllItemsAtStore.Keys)
```
- [x] Change the iteration variable identifier `item` to `itemId`
- [x] Change the iteration variable type to `ItemId`

DZ-FSDev commented on ColesStopAndShop/CashRegister.cs:
```diff
@@ -218,7 +218,15 @@ public void ProcessPayment(PaymentType paymentChosen)
+                    {
+                        if (itemScanned == item)
+                        {
+                            listOfItems.Add((ItemId)itemScanned);
```
- [x] Breaking out of the loop once a match is found will speed up execution for when the items sold dictionary grows

DZ-FSDev reviewed (changes requested):
**CR for #32**

CCianfloneDev committed:
Update CashRegister.cs

Changed variable type and identifier of item verification step for more clean code.

CCianfloneDev committed:
Update CashRegister.cs

Added break keyword in item verification step (foreach) to speed up execution time

DZ-FSDev reviewed (approved):
Looks good!
